cookbook/form/create_form_type_extension.rst

```diff
@@ -12,18 +12,10 @@ But sometimes, you don't really need to add new field types - you want
 to add features on top of existing types. This is where form type
 extensions come in.
 
-Form type extensions have 2 main use-cases:
-
-#. You want to add a **generic feature to several types** (such as
-   adding a "help" text to every field type);
-#. You want to add a **specific feature to a single type** (such
-   as adding a "download" feature to the "file" field type).
-
```

Comment:
I like these 2 bullet points - they quickly answer the question "what can I do with this form type extension thing?". I think you removed them because it says "every field type", but that's effectively true (the button being the exception). I'd like to have these back, and let the last paragraph explain the edge case.

What do you think?